benchmark/k8s.py

```diff
@@ -0,0 +1,74 @@
+from string import Template
+
+benchmark_pod_tmpl = '''
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: benchmark-sa
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: benchmark-privileged-psp
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: privileged-psp
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: benchmark-sa
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  # "namespace" omitted since ClusterRoles are not namespaced
+  name: benchmark-calico-resource-creator
+rules:
+- apiGroups: ["crd.projectcalico.org"]
+  #resources: ["*"]
+  resources: ["globalnetworkpolicies", "globalnetworksets","hostendpoints"]
+  verbs: ["get", "watch", "list", "create", "delete","update"]
+    #verbs: ["*"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: benchmark-calico-resource-creator
+roleRef:
+  kind: ClusterRole
+  name: benchmark-calico-resource-creator
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: benchmark-sa
+  namespace: default
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: egress-benchmark
+  labels:
+    app: benchmark
+spec:
```

Comment:
For the other tests, we use `docker run --net=host`. Could we have `hostNetwork: true`, in order to compare apples to apples?